packages/checkout/widgets-lib/src/widgets/sale/hooks/useSignOrder.ts

```diff
@@ -183,7 +183,7 @@ export const useSignOrder = (input: SignOrderInput) => {
       data: string,
       gasLimit: number,
       method: string,
-    ): Promise<string | undefined> => {
+    ): Promise<[error: any, hash: string | undefined]> => {
```

Comment:
non-blocking - normally i would expect the error to be the second option here as the hash is more important

Reply:
thank you, will do the fix.